src/main/java/frc/robot/subsystems/intake/IntakeIO.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.*;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.units.Voltage;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface IntakeIO {
+
+    IntakeInputsAutoLogged inputs = new IntakeInputsAutoLogged();
+
+    default void setAngle(MutableMeasure angle) {} // [Rad]
+
+    default void setRollerSpeed(MutableMeasure speed) {} // [Rps]
+
+    default void setCenterRoller(MutableMeasure speed) {}
```

Comment:
Shouldn't have default implementations, all of these methods will be implemented in all of the IOs.

Reply:
done.

Reply:
a2668c70dab52d8f6b7caac96cebb4f97cf1a4ae